app.py

```diff
@@ -13,10 +23,31 @@ def hello_world():
 def test():
     return 'Hello, test!'
 
-@app.route('/getData', methods = ['GET'])
-def getData():
-	return json.dumps({"success": True,
-  "events": [ {"chapter_number": 1, "section_number": 1, "user_id": 333, "date": datetime.datetime.now().isoformat()}, {"chapter_number": 1, "section_number": 1, "user_id": 444}]})
+def getEvents(zybook_code, user_id):
```

Comment:
Our convention for `Ember` code is camelCase. However, for python code we use lower case and dashes for all variables and method names. Ex: `get_events`.